README.md

```diff
@@ -1,7 +1,21 @@
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# Tvheadend HTSP PVR
-Tvheadend HTSP PVR client addon for [Kodi] (http://kodi.tv)
+# Tvheadend PVR addon for Kodi
+
+This is a [Kodi] (http://kodi.tv) PVR addon for connecting to a (tvheadend)[https://tvheadend.org] backend.
```

Comment:
You should use [tvheadend] followed by (https://tvheadend.org) or links look funny.
See https://raw.githubusercontent.com/xbmc/xbmc/master/README.md for reference